201-vnet-to-vnet-bgp/azuredeploy.json

```diff
@@ -35,8 +49,7 @@
         "vnet1Ref": "[resourceId('Microsoft.Network/virtualNetworks',variables('vNet1').name)]",
         "gateway1SubnetRef": "[concat(variables('vnet1Ref'),'/subnets/','GatewaySubnet')]",
         "vnet2Ref": "[resourceId('Microsoft.Network/virtualNetworks',variables('vNet2').name)]",
-        "gateway2SubnetRef": "[concat(variables('vnet2Ref'),'/subnets/','GatewaySubnet')]",    
-        "gatewaySku": "Standard"
+        "gateway2SubnetRef": "[concat(variables('vnet2Ref'),'/subnets/','GatewaySubnet')]"
```

Comment:
Can we update these to use resourceId instead of concat?  And remove the vnet*Ref variables if  they aren't used...